easybuild/easyconfigs/l/libX11/libX11-1.6.3-foss-2015b-Python-2.7.10.eb

```diff
@@ -0,0 +1,37 @@
+easyblock = 'ConfigureMake'
+
+name = 'libX11'
+version = '1.6.3'
+versionsuffix = '-Python-2.7.10'
```

Comment:
@verdurin the most recent easyconfigs for libX11 have been restructured to drop the dependency on Python, which is actually not really needed...